pkg/common/selector/selector.go

```diff
@@ -3,9 +3,16 @@
 package selector
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/spiffe/spire/proto/spire/common"
 )
 
+// Type and Value are delimited by a colon (:)
+// e.g. "unix:uid:1000"
+const Delimiter = ":"
```

Comment:
After this PR is merged, I will create another PR to use this constant in the SPIRE code.